### PR TITLE
Bump version to 1.1.4. Customize user agent

### PR DESCRIPTION
**Pull request by dima-ko**

### CODE REVIEW

File: android-client/src/main/java/io/msgs/v2/MsgsClient.java [hunk 1]
```diff
@@ -141,6 +149,9 @@ public EndpointRequestHelper forEndpoint(String token) {
     private Map<String, String> _getApiHeader() {
         Map<String, String> headers = new HashMap<>();
         headers.put("X-MsgsIo-APIKey", _apiKey);
+        if (_customUserAgent != null) {
+            headers.put("User-Agent", _apiKey);
```
Comment (dzolnai) [inline android-client/src/main/java/io/msgs/v2/MsgsClient.java]: Wrong value

**Review by dzolnai — CHANGES_REQUESTED** [review]

File: android-client/src/main/java/io/msgs/v2/MsgsClient.java (hunk 1 above)
Comment (dima-ko) [reply android-client/src/main/java/io/msgs/v2/MsgsClient.java]: facepalm.jpg

**Comment by dima-ko** [comment]
fix on comment

File: android-client/src/main/java/io/msgs/v2/MsgsClient.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 
     private final String _baseURL;
     private final String _apiKey;
+    private String _customUserAgent;
```
Comment (dzolnai) [inline android-client/src/main/java/io/msgs/v2/MsgsClient.java]: Make this final

**Review by dzolnai — APPROVED** [review]

**Pull request merged**